packages/docs/components-listview.md

```diff
@@ -0,0 +1,51 @@
+---
+id: components-listview
+title: ListView
+sidebar_label: ListView
+---
+
+## ListView
+
+List View is a special type of view that can display items one by one based in a list.
+
+```JS
+const DATA = [
+  {
+    title: 'Main dishes',
+    data: ['Pizza', 'Burger', 'Risotto'],
+  },
+  {
+    title: 'Sides',
+    data: ['French Fries', 'Onion Rings', 'Fried Shrimps'],
+  },
+  {
+    title: 'Drinks',
+    data: ['Water', 'Coke', 'Beer'],
+  },
+  {
+    title: 'Desserts',
+    data: ['Cheese Cake', 'Ice Cream'],
+  },
+];
+
+function Item({ title }) {
+  return (
+    <View style={styles.item}>
+      <Text style={styles.title}>{title}</Text>
+    </View>
+  );
+}
+
+export default function App() {
+  return (
+      <SectionList
```

Comment:
I don't think we have `ListView` working that way yet and should be `ListView` instead of `SectionList`

Reply:
Fixed!